src/sInv.sol

```diff
@@ -88,17 +98,21 @@ contract sINV is ERC4626 {
     }
 
     modifier onlyGov() {
-        require(msg.sender == gov, "ONLY GOV");
+        if(msg.sender != gov) revert OnlyGov();
+        _;
+    }
+
+    modifier onlyPendingGov() {
+        if(msg.sender != pendingGov) revert OnlyPendingGov();
         _;
     }
 
     /**
      * @dev Hook that is called after tokens are deposited into the contract.
-     * @param assets The amount of assets that were deposited.
      */    
-    function afterDeposit(uint256 assets, uint256) internal override {
-        require(totalSupply >= MIN_SHARES, "Shares below MIN_SHARES");
-        uint invBal = asset.balanceOf(address(this));
+    function afterDeposit(uint256, uint256) internal override {
+        if(totalSupply < MIN_SHARES) revert BelowMinShares();
+        uint256 invBal = asset.balanceOf(address(this));
         if(invBal > minBuffer){
             asset.transfer(address(invEscrow), invBal - minBuffer);
```

Comment:
I don't think it's an issue, as Deposit events are meant primarily for tracking user events and the sINV contract is a bit different from a normal user.
Ultimately the `market.deposit(...)` function isn't that much more gas intensive, so it may be worth it to deposit the funds normally. I'll have a conversation with the team about it.